yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -62,10 +65,11 @@ module vyatta-protocols-frr-isis-v1 {
     SPDX-License-Identifier: BSD-3-Clause
     The YANG module package for vyatta-protocols-frr-isis-v1";
 
-  revision 2020-11-17 {
+  revision 2020-12-18 {
```

Comment:
Presumably the 2020-11-17 statement has not yet shipped?

Reply:
No, it hasn't.